### ISSUE TITLE
Parameter hints and quick view are not working for any package apart from Base and Core

### ISSUE THREAD

**Issue by ColinMoldenhauer** [julia-vscode/julia-vscode#3454]
Hi everyone,

I am new to VS Code, but after some googling it seems like many people encounter this problem.
Usually, `ctrl + q` would trigger a quick view about the current function, and `ctrl + p ` will show parameter hints. This also works fine for the `Base` and `Core` modules, but not for other inbuilds like `Plots` or `DifferentialEquations`.

This problem appears to affect a very fundamental feature of modern IDEs, so I am wondering whether there is some way to fix this issue?

**Comment by dpinol** [comment]
Yes, the "meat" should be already implemented because the hint already appears when opening the parameters bracket or typing a parameters comma. 
Selecting "Trigger parameter hints" from "Command palette" does not work either 
https://stackoverflow.com/questions/53731271/how-to-trigger-parameter-hints-in-visual-studio-code

**Comment by pfitzseb** [comment]
Not sure what's going on here, but you should be getting those hints if said packages are in your current environment:
![image](https://github.com/julia-vscode/julia-vscode/assets/6735977/a6472199-fb37-4ae8-9ecc-912b7dec50a1)

There seems to be a specific issue with `Plots.plot` though where we don't even know about any methods defined for it.

**Comment by dpinol** [comment]
> Not sure what's going on here, but you should be getting those hints if said packages are in your current environment:

You're right, it's working now on my project. Maybe if I hadn't correctly selected the environment before
thanks!

**Comment by davidanthoff** [comment]
The language server backend was rewritten in v1.220.0: StaticLint.jl and SymbolServer.jl were folded into JuliaWorkspaces, and completion, hover, signature help, go-to-definition and find-references were reimplemented. This report predates that rewrite, so the code that produced it no longer exists and the behaviour cannot be confirmed against the current engine.

Closing as outdated. If you still see this on a current release, please open a fresh issue with a reproducer and we will look at it against the new engine.
